### DISCUSSION TITLE
Welcome to Azure AD Application Management Discussions!

### DISCUSSION THREAD

**Discussion by LockTar** [LockTar/AzureAdApplicationManagement#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by staycoolkhan** [comment]
Hi Lock, First of all thanks for this important repository. While fallowing your ReadMe doc to setup the azure devops pipeline I'm facing this issue **"##[error]Resource not found for the segment 'me'.**" in the pipeline. Please could you help me. Thanks
![image](https://user-images.githubusercontent.com/25347487/230753145-4fdc673c-ac07-4403-ad6b-5b866d4b5d01.png)

**Reply by LockTar** [reply]
Hi,
I think you havent followed the getting started completely. Are the permissions set and do they have an admin consent? Check if you have really set application permissions and not delegated. Check the permissions depending on the version of the tasks.

If it is an existing application, set the owner. Follow faq if you don't know how.

**Reply by LockTar** [reply]
The best thing is to use v4 of the tasks. Almost the same as v3. Samples will be updated shortly

**Reply by staycoolkhan** [reply]
Thanks for the help. It is now resolved by updating to v4.

**Reply by LockTar** [reply]
Good to hear!

**Reply by staycoolkhan** [reply]
Hi LockTar, I have now another issue with [secre](secrets: '[{ ''Description'': ''testkey'', ''EndDate'': ''01/12/2024'' }]')
![image](https://user-images.githubusercontent.com/25347487/230771663-db34741c-dd15-45ce-96da-98a2898dfa0d.png)

**Reply by staycoolkhan** [reply]
This is also resolved by passing [''endDate"": ](secrets: '[{ ''Description'': ''testkey'', ''endDate'': ''2025-02-24'' }]'). Thanks
